debian/control

```diff
@@ -162,7 +147,7 @@ Depends: ${shlibs:Depends},
 Provides: nymea-update-plugin-api-1,
           nymea-zeroconf-plugin-api-1,
           nymea-system-plugin-api-3
-Description: An open source IoT server - core library
+Description: An open source IoT server - library
```

Comment:
Perhaps make it "nymea plugin development API library"

Reply:
will update in #546